### ISSUE TITLE
修改完以后使用cornerStone对比代码崩溃

### ISSUE THREAD

**Issue by kingsword** [yulingtianxia/pbxprojHelper#10]
看了看源码终于知道怎么使用了，但是修改之后使用CornerStone对比代码的时候，导致CornerStone崩溃，猜测是编码的问题，可以解决一下么？

**Comment by yulingtianxia** [comment]
@kingsword 可以把你的工程文件和 json 文件发下么

**Comment by kingsword** [comment]
@yulingtianxia 
[pbxproj.zip](https://github.com/yulingtianxia/pbxprojHelper/files/1817779/pbxproj.zip)

**Comment by yulingtianxia** [comment]
@kingsword 我看你发的工程文件是xml 格式，不是原始的工程文件。我猜测是不是因为 cornerstone 把 xml 格式跟原始工程比对后，差距十分巨大，然后 crash 了呢？我也用 cornerstone，老是出问题。

**Comment by kingsword** [comment]
对 我发送的是修改后的文件，我还真没注意修改后变成XML格式了。具体也不太清楚为啥这样，反正就是崩溃，不过目前影响也不大，一般要上线的时候才去修改工程文件~

**Comment by frankKiwi** [comment]
工程配置随便勾一下就行了
